cfme/tests/cloud/test_cloud_timelines.py

```diff
@@ -22,7 +23,7 @@
 def delete_fx_provider_event(db, provider):
     logger.info("Deleting timeline events for provider name {}".format(provider.name))
     ems = db['ext_management_systems']
-    ems_events = db['ems_events']
+    ems_events = version.pick({version.LOWEST: db['ems_events'], '5.5': db['event_streams']})
```

Comment:
This needs to be like this

```python
ems_events_table_name = version.pick({version.LOWEST: 'ems_events', '5.5': 'event_streams'})
ems_events = db[ems_events_table_name]
```
The calls to ```db['ems_events'] ``` were not being deferred and hence failing in PRT